src/cloudkit.cs

```diff
@@ -2039,4 +2039,41 @@ interface CKFetchDatabaseChangesOperation
 		[NullAllowed, Export ("recordZoneWithIDWasDeletedDueToUserEncryptedDataResetBlock", ArgumentSemantic.Copy)]
 		Action<CKRecordZoneID> RecordZoneWithIdWasDeletedDueToUserEncryptedDataReset { get; set; }
 	}
+
+	[TV (16,0), NoWatch, Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (NSObject))]
+	interface CKAllowedSharingOptions : NSSecureCoding, NSCopying
```

Comment:
I think this needs a `[DisableDefaultCtor]` since there is a native ctor 

```suggestion
	[DisableDefaultCtor]
	interface CKAllowedSharingOptions : NSSecureCoding, NSCopying
```